nailgun/entities.py

```diff
@@ -1018,7 +1018,7 @@ def create(self, create_missing=None):
         <https://bugzilla.redhat.com/show_bug.cgi?id=1223540>`_.
 
         """
-        return AbstractDockerContainer(
+        return self.__class__(
```

Comment:
Subclasses will inherit this `create()` func and use `AbstractDockerContainer().read()`, so they wan't receive extra fields specified in `__init__()`, for example. This can be resolved by implementing separate create() method for each subclass... or this way :)

Before:
```python
>>> container = entities.DockerHubContainer(
                    command='top',
                    compute_resource=compute_resource,
                    organization=[self.org],
                    repository_name=repo.container_repository_name,
                    tag='latest',
                    tty='yes',
                ).create()
>>> 'tag' in container._fields
False
>>> 'repository_name' in container._fields
False
```

After:
```python
>>> container = entities.DockerHubContainer(
                    command='top',
                    compute_resource=compute_resource,
                    organization=[self.org],
                    repository_name=repo.container_repository_name,
                    tag='latest',
                    tty='yes',
                ).create()
>>> 'tag' in container._fields
True
>>> 'repository_name' in container._fields
True
>>> container.tag
u'latest'
>>> container.repository_name
u'36e00d5a-165a-4d80-a137-8428da9c3d4a-3054e666-2944-455b-bc6a-f0f0c69bcfda-ew0vn3ssv8adidx'
```